CONTRIBUTING.md

```diff
@@ -1,7 +1,24 @@
 # Contribution Guidelines for the Firefox Accounts Content Server
```

Comment:
Is this really still content server specific?

Reply:
thanks, fixed